common/src/main/java/org/openmetadata/annotations/ExposedAnnotator.java

```diff
@@ -59,8 +61,12 @@ private void addMaskedFieldAnnotationIfApplies(JMethod jMethod, String propertyN
       Field outerClassField = JMethod.class.getDeclaredField("outer");
       outerClassField.setAccessible(true);
       JDefinedClass outerClass = (JDefinedClass) outerClassField.get(jMethod);
-      if (outerClass.fields().containsKey(propertyName)
-          && outerClass.fields().get(propertyName).annotations().stream()
+
+      TreeMap<String, JFieldVar> insensitiveFieldsMap = new TreeMap<>(String.CASE_INSENSITIVE_ORDER);
+      insensitiveFieldsMap.putAll(outerClass.fields());
```

Comment:
we had to apply a case-insensitive map here otherwise the `jwtToken` was not getting properly annotated with `Password`.

The issue was that the field is `jWTToken` on the generated class, and the setter/getters did not directly contain the `jWTToken` string in the name as they're called `getJWTToken` & `setJWTToken`. Checking without cases handles properly the starting lowercase `j` in this scenarios